src/components/rtoken-setup/governance/GovernanceParameters.tsx

```diff
@@ -12,8 +12,8 @@ const GovernanceParameters = (props: BoxProps) => {
     'votingPeriod',
     'minDelay',
   ])
-  const votingDelayHelper = parseDuration(Number(votingDelay) || 0 * 12)
-  const votingPeriodHelper = parseDuration(Number(votingPeriod) || 0 * 12)
+  const votingDelayHelper = parseDuration(Number(votingDelay) * 12 || 0)
```

Comment:
I'm not sure this part is right, because I'm not sure what it does in the FormField. If this is converting the number in the field to seconds, and then that's rendering using some sort of datetime lib, then that's correct. If this is saying that the number sent to the backend should be 12x more than the number input in the FormField, then it's a bug.

Reply:
parsing blocks into seconds for getting the 3 days.. string from datelib

Reply:
Looks good, then

Reply:
It's the former - so it should be correct now (where it wasn't before) 
![image](https://user-images.githubusercontent.com/71284258/233165118-02b10047-8ff2-4a80-b711-5a481f2f4218.png)